### ISSUE TITLE
[FEATURE] Show if a version is in use

### ISSUE THREAD

**Issue by Mossy93** [Vysp3r/ProtonPlus#147]
**Is your feature request related to a problem? Please describe.**
I would like to see if a particular version is in use, just like we can do on ProtonUp-QT

**Describe the solution you'd like**
Show if a version is in use, and maybe include an option to show games using it.

**Additional context**
ProtonUp-QT has this feature.

**Comment by Vysp3r** [comment]
Yeah, I am currently working on the 'Games' tab just like ProtonUp-Qt have. It will be available in the v0.5. I will make sure to add your request at the same time since doing that tab is a requirement for your request.

**Comment by Majudy** [comment]
Any updates on this?

**Comment by Vysp3r** [comment]
> Any updates on this?

I will not be available for two months for personal reasons so don't expect this soon. 
Maybe I will have time to do it in December, but not before that.
May I ask why would it be useful for you?

**Comment by Majudy** [comment]
No rush man you're doing great work and I hope all is well, this feature would be useful to me because I don't like to keep old versions of proton unless a game doesn't support a newer version so I typically delete all old versions and keep one version installed. In ProtonUp-QT I can see my proton games and what version they're using which allows me to quickly update them all to a new version instead of manually sifting through my library and changing the proton version for all my proton games to a new one. I prefer your software though so as soon as this feature us implemented I can remove ProtonUp-QT from my system.

**Comment by Vysp3r** [comment]
Alright, I'll keep that in my notes for when I come back

**Comment by Vysp3r** [comment]
I am working on the Steam integration right now which will contain your feature.
You can follow here: https://github.com/Vysp3r/ProtonPlus/issues/326#issuecomment-2899557709
